internal/backends/winit/renderer/sw.rs

```diff
@@ -106,20 +123,85 @@ impl<const BUFFER_COUNT: usize> super::WinitCompatibleRenderer for SoftwareRende
         canvas.opengl_context.make_not_current();
     }
 
-    fn default_font_size() -> LogicalLength {
-        i_slint_core::software_renderer::SoftwareRenderer::<BUFFER_COUNT>::default_font_size()
+    fn resize_event(&self, size: PhysicalWindowSize) {
+        let canvas = if self.canvas.borrow().is_some() {
+            std::cell::Ref::map(self.canvas.borrow(), |canvas_opt| canvas_opt.as_ref().unwrap())
+        } else {
+            return;
+        };
+
+        canvas.opengl_context.ensure_resized(size)
     }
 }
 
-pub(crate) struct SwCanvas {
-    canvas: RefCell<femtovg::Canvas<femtovg::renderer::OpenGl>>,
-    opengl_context: crate::OpenGLContext,
-}
+impl<const MAX_BUFFER_AGE: usize> Renderer for SoftwareRenderer<MAX_BUFFER_AGE> {
```

Comment:
I hope this is temporary.  It's a bit sad to just forward everything.

Instead of being a Renderer itself,  WinitCompatibleRenderer could have a `fn renderer(&self)->&dyn Renderer`   accessor.

Reply:
Oh that’s a good idea. Thanks, I’ll do that.